### ISSUE TITLE
Create tests for azureedge URIs

### ISSUE THREAD

**Issue by garvinhicking** [TYPO3-Documentation/render-guides#245]
Add the ability to set the env vars that are evaluated on GitHub actions for using azureedge URIs, to see their absolute URIs are properly built.

Needed for commits like https://github.com/TYPO3-Documentation/render-guides/pull/242/files to ensure no BC occurs.

**Comment by CybotTM** [comment]
@garvinhicking this is obsolete, right?

**Comment by garvinhicking** [comment]
Yes you're right, i think this is covered with the functional tests. Closing!
